frontend/vite.config.ts

```diff
@@ -14,10 +17,60 @@ export default defineConfig(({ mode }) => {
     return {
         plugins: [
             react(),
+            mdx({
```

Comment:
I really don't like introducing all these new dependencies, I just think we could have done it better and faster without